packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -487,6 +485,47 @@ class _TextSelectionHandleOverlay extends StatefulWidget {
 class _TextSelectionHandleOverlayState extends State<_TextSelectionHandleOverlay> {
   Offset _dragPosition;
 
+  static const Duration _fadeDuration = Duration(milliseconds: 150);
+  AnimationController _controller;
+  Animation<double> get _opacity => _controller.view;
+
+  ValueListenable<bool> get _visibility {
+    switch (widget.position) {
+      case _TextSelectionHandlePosition.start:
+        return widget.renderObject.selectionStartInView;
+      case _TextSelectionHandlePosition.end:
+        return widget.renderObject.selectionEndInView;
+    }
+    return null;
+  }
+
+  @override
+  void initState() {
+    super.initState();
+
+    _controller = AnimationController(duration: _fadeDuration, vsync: widget.vsync)..value = 0.0;
```

Comment:
value should start at 0.0, no need to set it

Reply:
Done.